src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/AddToolbarAction.js

```diff
@@ -8,7 +8,7 @@ export default class AddToolbarAction extends AbstractToolbarAction {
         return {
             icon: 'su-plus-circle',
             label: translate('sulu_admin.add'),
-            onClick: action(this.list.addItem),
+            onClick: action(this.list.handleItemAdd),
```

Comment:
That's actually the reason it is not called `handleItemAdd`. It is an external API as well, in which case the `handle` part makes no sense...

Reply:
kind of feeling stupid now, but what is the reason? 🙈

Reply:
The `addItem` method of the `List` view is part of its public API, meaning that anybody who gets access to its ref can call this method. And if you think about other objects/classes, you also wouldn't prefix every method with `handle`, or would you?

Reply:
alright, that makes sense. reverted this change :)